telemetry/specs/INT.mdk

```diff
@@ -687,17 +687,17 @@ order of bits set in the instruction bitmap.
     either add corresponding metadata filled with the reserved value 0xFFFF_FFFF
     or must not add any INT metadata to the packet.
   - If an INT transit hop does not add metadata to a packet due to any of the
-    above reasons, it must not increment the total INT hop count in the INT
+    above reasons, it must not decrement the remaining INT hop count in the INT
     metadata header.
 * Summary of the field usage
   - The INT Source must set the following fields:
-    - Ver, Rep, C, Instruction Count, Max Hop Count, Total Hop Count (0), and
-Instruction Bitmap.
+    - Ver, Rep, C, Instruction Count, Remaining Hop Count, and Instruction
+      Bitmap.
     - INT Source must set all reserved bits to zero.
   - Intermediate devices can set the following fields:
-    - C, E, Total Hop Count
+    - C, E, Remaining Hop Count
 * In an INT packet, the length (in bytes) of the INT metadata stack must always
-be (Instruction Count \* 4 \* Total Hop Count).
+be a multiple of (Instruction Count \* 4).
```

Comment:
Shall we clarify how INT Sink can derive the total metadata bytes from the 'length' field in shim header? Every encap -- INT over TCP/UDP, vxlan-gpe, and geneve -- has shim or option header, which includes the total word length. The sink can compute 
   metadata stack size = 'length' * 4 - [total fixed header sizes]
The total fixed header sizes must be known for each encap.

Reply:
Agree, let's add some clarification on how INT sink can derive total metadata length.

Reply:
Done